### ISSUE TITLE
Add macros for task delay to Tasks.h

### ISSUE THREAD

**Issue by IshDeshpa** [lhr-solar/Controls#336]
Currently, most if not all of our tasks have a main while loop with a delay. It would be nice if all of these delays were macros, and all were defined in Tasks.h (at least for the main application running tasks).

**Comment by babusid** [comment]
Why though, it's literally a 1-line function call with a super-simple API? Unless the purpose is to aid in the micrium->frtos migration.

**Comment by IshDeshpa** [comment]
I don't think you understand. The macro is not for the function call, but for the number of ms to delay for each task's main while loop. This is to eliminate magic numbers.

**Comment by babusid** [comment]
Ah makes more sense
